sieve/bc_eval.c

```diff
@@ -2055,7 +2056,18 @@ int sieve_eval_bc(sieve_execute_t *exe, int *impl_keep_p, sieve_interp_t *i,
                 message = parse_string(message, variables);
             }
 
-            res = do_notify(notify_list, cmd.u.n.id, cmd.u.n.from, cmd.u.n.method,
+            if (op == B_ENOTIFY) {
+                /* Parse/split the method URI */
+                if (!strncmp(method, "mailto:", 7)) {
```

Comment:
should that be `strcasecmp` or does the Sieve spec require the URI scheme to be lower-case?

Reply:
Yeah, case-insensitive makes sense.